<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTimeBase</class>
 <widget class="QDialog" name="DlgTimeBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>时基表</string>
  </property>
  <widget class="QPushButton" name="butfind">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>查 找</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butset">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>设 置</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>51</y>
     <width>341</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;����&quot;;</string>
   </property>
   <property name="title">
    <string>调度日</string>
   </property>
   <widget class="QCheckBox" name="cd1_31">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>216</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>31</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_18">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>182</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QCheckBox" name="cd1_26">
     <property name="text">
      <string>26</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_27">
     <property name="text">
      <string>27</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_28">
     <property name="text">
      <string>28</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_29">
     <property name="text">
      <string>29</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_30">
     <property name="text">
      <string>30</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_13">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>23</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QCheckBox" name="cd1_1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_4">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_5">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_14">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>53</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QCheckBox" name="cd1_6">
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_7">
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_8">
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_9">
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_10">
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_16">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>117</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QCheckBox" name="cd1_16">
     <property name="text">
      <string>16</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_17">
     <property name="text">
      <string>17</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_18">
     <property name="text">
      <string>18</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_19">
     <property name="text">
      <string>19</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_20">
     <property name="text">
      <string>20</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_15">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>85</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QCheckBox" name="cd1_11">
     <property name="text">
      <string>11</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_12">
     <property name="text">
      <string>12</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_13">
     <property name="text">
      <string>13</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_14">
     <property name="text">
      <string>14</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_15">
     <property name="text">
      <string>15</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_17">
    <property name="geometry">
     <rect>
      <x>44</x>
      <y>148</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QCheckBox" name="cd1_21">
     <property name="text">
      <string>21</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_22">
     <property name="text">
      <string>22</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_23">
     <property name="text">
      <string>23</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_24">
     <property name="text">
      <string>24</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cd1_25">
     <property name="text">
      <string>25</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="checkBoxifofday">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>214</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>全选</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>42</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;����&quot;;</string>
   </property>
   <property name="text">
    <string>调度计划号</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>256</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;����&quot;;</string>
   </property>
   <property name="title">
    <string>调度周</string>
   </property>
   <widget class="QCheckBox" name="cw1_1">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>28</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>周一</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cw1_2">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>28</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>周二</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cw1_3">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>28</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>周三</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cw1_4">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>28</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>周四</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cw1_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>61</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>周五</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cw1_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>61</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>周六</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cw1_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>61</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>周日</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxifofweek">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>全选</string>
    </property>
   </widget>
  </widget>
  <widget class="QSpinBox" name="spinNumber">
   <property name="geometry">
    <rect>
     <x>276</x>
     <y>42</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinTimeTab">
   <property name="geometry">
    <rect>
     <x>536</x>
     <y>325</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>431</x>
     <y>325</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;����&quot;;</string>
   </property>
   <property name="text">
    <string>时段表号</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>75</y>
     <width>341</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;����&quot;;</string>
   </property>
   <property name="title">
    <string>调度月</string>
   </property>
   <widget class="QCheckBox" name="cm1_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>33</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cm1_10">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>110</y>
      <width>38</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cm1_11">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>110</y>
      <width>38</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>11</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cm1_12">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>110</y>
      <width>38</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>12</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cm1_5">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>67</y>
      <width>29</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cm1_6">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>67</y>
      <width>29</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cm1_7">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>68</y>
      <width>29</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cm1_8">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>68</y>
      <width>29</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cm1_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>26</y>
      <width>29</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cm1_2">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>26</y>
      <width>29</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cm1_3">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>27</y>
      <width>29</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cm1_4">
    <property name="geometry">
     <rect>
      <x>264</x>
      <y>28</y>
      <width>29</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxifofmon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>146</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>全选</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="bufclear">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>清空行</string>
   </property>
  </widget>
  <widget class="QLabel" name="hang">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>332</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="butquery">
   <property name="geometry">
    <rect>
     <x>656</x>
     <y>319</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>查询
已设</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butclearall">
   <property name="geometry">
    <rect>
     <x>716</x>
     <y>319</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>清空
全表</string>
   </property>
  </widget>
  <widget class="QLabel" name="display">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>365</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
